examples/by_feature/fsdp_with_peak_mem_tracking.py

```diff
@@ -293,7 +294,7 @@ def collate_fn(examples):
                     {
                         "accuracy": eval_metric["accuracy"],
                         "f1": eval_metric["f1"],
-                        "train_loss": total_loss,
+                        "train_loss": total_loss.item() if type(total_loss) == torch.Tensor else total_loss,
```

Comment:
Do we really have either a Tensor or something else? Can't it always be `.item()` here?